SPECS/bmake/bmake.spec

```diff
@@ -44,6 +45,9 @@ sh ./make-bootstrap.sh
 ./bmake -m mk install DESTDIR=%{buildroot} INSTALL='install -p' STRIP_FLAG=''
 chmod a-x %{buildroot}%{_datadir}/mk/mkopt.sh
 
+%check
+./bmake -m mk test
```

Comment:
Note: I don't think we've seen the random failures of the `job-output-null` test in 3.0. So I won't disable that test as I did in the 2.0 package [here](https://github.com/microsoft/azurelinux/pull/13810)